intentv1/intent.ridl

```diff
@@ -0,0 +1,103 @@
+webrpc = v1
+
+name = sequence-waas-intents
+version = v0.1.0
+
+struct Signature
+  - sessionId: string
+  - signature: string
+
+# no way to generate string enums
+# enum IntentName: uint8
+#  - openSession
+#  - closeSession
+#  - validateSession
+#  - finishValidateSession
+#  - listSessions
+#  - getSession
+#  - sign
+#  - transaction
+
+struct Intent
+  - version: string
+  - name: string
+  - expires: uint64
+  - issued: uint64
+  - data: any
+  - signatures: []Signature
+
+struct SessionPacketProof
```

Comment:
"Packet" .. nooooooooo

what is this struct for btw....? (so I can help suggest another name)

Reply:
lets get rid of this type.. move fields to remove the nesting

Reply:
done